### PR TITLE
config/settings: add assets in staticfiles dirs

### PR DESCRIPTION
**Pull request by m4ra**

### CODE REVIEW

File: meinberlin/config/settings/base.py [hunk 1]
```diff
@@ -207,6 +207,7 @@
 
 STATICFILES_DIRS = [
     os.path.join(PROJECT_DIR, "static"),
+    os.path.join(PROJECT_DIR, "assets"),
```
Comment (philli-m) [inline meinberlin/config/settings/base.py:210]: @m4ra so looking at this made me think it actually might be a webpack issue (sorry!!) as webpack should add the files to static, I kind of blanked on that as I thought it was just not collecting the new static files but if they are never in static then webpack has failed to add them, I'll have a look after lunch.

Comment (m4ra) [reply meinberlin/config/settings/base.py:210]: there are some other files that are missing from static folders. how does webpack work? Maybe it's fine to have the assets too, for local development.

Comment (philli-m) [reply meinberlin/config/settings/base.py:210]: we use something called CopyWebpackPlugin that should copy over all image files from the asset folder into the defined output folder which is static which is why we shouldn't need to add assets to our settings, i'm not sure of the advantages to using webpack over static in these cases though but as we bundle our other assets with webpack, it's probably more consistent?

Comment (goapunk) [reply meinberlin/config/settings/base.py:210]: @m4ra during the build on the build server we call `make release`, which in turn runs `npm run build:prod`. Locally I think it happens with `make watch` but don't know from the top of my head

**Comment by m4ra** [comment]
Locally yes, I've seen it in make watch.
Closing this PR.

**Pull request closed without merging**